src/sentry/incidents/models/alert_rule.py

```diff
@@ -225,7 +225,7 @@ class Meta:
         unique_together = (("alert_rule", "project"),)
 
 
-class AlertRuleMonitorType(Enum):
+class AlertRuleMonitorType(IntEnum):
```

Comment:
👍 nice

Can we actually append `Int` to the enum name?

As we transition over to `IntEnum`'s, i think it would make it more clear why some of our enums are being `.value`'d whereas others are being compared directly 😅